src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -298,10 +298,14 @@ void WgRenderDataShape::updateMeshes(WgContext &context, const RenderShape &rsha
         } else if (cmd == PathCommand::Close) {
             polyline.close();
         } else if (cmd == PathCommand::CubicTo) {
+            WgPoint p0 = polyline.pts.last();
+            WgPoint p3 = rshape.path.pts[pntIndex + 2];
+            size_t nsegs = (uint32_t)(p0.trans(rt->m).dist(p3.trans(rt->m)));
```

Comment:
I may not understand something, so please correct me
in a case, when `p0` and `p3` are so close to each other, that `dist < 1`, but `p1` and `p2` are quite far away:
<img width="400" alt="Zrzut ekranu 2024-06-19 o 18 22 09" src="https://github.com/thorvg/thorvg/assets/67589014/b25dd594-d32e-433d-bcd9-347ed08de1d2">
will the result still be as we expect it to be?

Reply:
Did you have some proposal about this? How to estimate total length of spline? As I undestand there is no general solution about it, but we can play around with p0, p1, p2 and p3 (green lines). But as I can see distance between p0 and p3 is more than OK

Reply:
there is no formula, we can only estimate. I'm just asking whether your estimation is good enough and won't give wrong results in such a case, when distance between p0 and p3 is very small. the first approach in estimating the bezier curve length is the sum of distances between all four points p0-p3

Reply:
Thats easy to implement. Lets try